scripts/e2e-helper.py

```
@@ -0,0 +1,325 @@
+                        if not hmac.compare_digest(f_mac_check, f_mac):
+                            continue
+                    cipher2 = Cipher(algorithms.AES(enc_key), modes.CBC(f_iv), backend=default_backend())
+                    dec = cipher2.decryptor()
+                    name_plain = dec.update(f_ct) + dec.finalize()
+                    unpadder = padding.PKCS7(128).unpadder()
+                    plain_name = (unpadder.update(name_plain) + unpadder.finalize()).decode()
+                    if plain_name == "data-key":
+                        f["value"] = encrypt_cipher_field("modified-value", enc_key, mac_key)
+                        break
+                except Exception:
+                    pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)